### ISSUE TITLE
BUG: Composer does not install correctly the v1.19.7 of SimpleSAMLphp

### ISSUE THREAD

**Issue by malavolti** [simplesamlphp/simplesamlphp#1761]
# BUG: Composer does not install correctly the v1.19.7 of SimpleSAMLphp

**Describe the bug**
Missing modules if SimpleSAMLphp 1.19.7 is installae with Composer

**To Reproduce**
1. Download & Install Composer: 
    * `wget "https://getcomposer.org/installer" -O /usr/local/src/composer-setup.php`
    * `php /usr/local/src/composer-setup.php --install-dir=/usr/local/bin --filename=composer`
2. Create the required directories:
    * `mkdir -p /var/simplesamlphp/cert /var/simplesamlphp/log /var/simplesamlphp/data`
3. Install SimpleSAMLphp with Composer:
    * `cd /var/simplesamlphp`
    *  `composer require --prefer-stable simplesamlphp/simplesamlphp`
5. I obtain the following error when I try an authentication:
    ```
    SimpleSAML\Error\Error: UNHANDLEDEXCEPTION

    Backtrace:
    1 www/_include.php:17 (SimpleSAML_exception_handler)
    0 [builtin] (N/A)
    Caused by: Exception: No module named 'ldap' has been installed.
    Backtrace:
    7 lib/SimpleSAML/Module.php:441 (SimpleSAML\Module::resolveClass)
    6 lib/SimpleSAML/Auth/Source.php:335 (SimpleSAML\Auth\Source::parseAuthSource)
    5 lib/SimpleSAML/Auth/Source.php:381 (SimpleSAML\Auth\Source::getById)
    4 lib/SimpleSAML/Auth/Simple.php:68 (SimpleSAML\Auth\Simple::getAuthSource)
    3 lib/SimpleSAML/Auth/Simple.php:168 (SimpleSAML\Auth\Simple::login)
    2 modules/core/www/authenticate.php:38 (require)
    1 lib/SimpleSAML/Module.php:266 (SimpleSAML\Module::process)
    0 www/module.php:10 (N/A)
    ```
    because SimpleSAMLphp does not find the "ldap" module into `/var/simplesamlphp/vendor/simplesamlphp/simplesamlphp/modules`.

6. Into `/var/simplesamlphp/vendor/simplesamlphp` I find only the following modules:
    ````
    admin
    core
    cron
    exampleauth
    multiauth
    portal
    saml
    ````
    and my SimpleSAMLphp doesn't work anymore. It works if I install the 1.19.5 version of SimpleSAMLphp.
    
I think there is a bug on the installation through Composer.

Thank you so much!
Marco

**Comment by tvdijen** [comment]
Hi @malavolti ! Yes, this is related to an issue with the `composer-module-installer`. It has been solved in 2.0-rc3 and I see no easy way to fix this in 1.19 without having to release another dot-version

**Comment by tvdijen** [comment]
Related to https://github.com/simplesamlphp/composer-module-installer/issues/23

**Comment by tvdijen** [comment]
@malavolti I think this was resolved now!
I tested with `composer require --prefer-stable simplesamlphp/simplesamlphp:^1.19` because in the mean time we have released a new 2.0.0 stable..  Could you please verify that the issue was resolved?
